### PR TITLE
Add profile section to "Installing IML On Vagrant"

### LINKED ISSUE #110: SCD - Add profile selection to "Installing IML On Vagrant"
The profile selection details were written in the ZFS and LDISKFS monitored only documents. This information has been removed in favor of using a provision script so it needs to be added to the "Installing IML On Vagrant" page.

### PR DESCRIPTION
**Pull request by johnsonw**
Fixes #110.

- Add profile section to Installing IML On Vagrant
- Extract profiles list into template

Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

File: docs/Contributor_Docs/cd_Installing_IML_On_Vagrant.md [hunk 1]
```diff
@@ -65,13 +65,21 @@ Please refer to [https://github.com/whamcloud/vagrantfiles](https://github.com/w
 
 ## Adding Servers
 
-You should now be able to see IML when navigating to [https://localhost:8443](https://localhost:8443). Click on the login link at the top right and log in as the admin user (password: lustre). Next, go to the server configuration page and add the following servers:
+You should now be able to see IML when navigating to [https://localhost:8443](https://localhost:8443). Click on the login link at the top right and log in as the admin user (password: lustre). After logging in, go to the servers page by selecting the following:
+
+> Configuratin -> Servers
```
Comment (jgrund) [inline docs/Contributor_Docs/cd_Installing_IML_On_Vagrant.md]: typo

File: docs/Contributor_Docs/cd_Installing_IML_On_Vagrant.md [hunk 2]
```diff
@@ -65,13 +65,21 @@ Please refer to [https://github.com/whamcloud/vagrantfiles](https://github.com/w
...
+
+Click the `Add Servers` button and enter the following pdsh expression:
```
Comment (jgrund) [inline docs/Contributor_Docs/cd_Installing_IML_On_Vagrant.md]: pdsh -> hostlist

File: _includes/profile-list.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{{ include.whitespace }}- **Monitored Storage Server:** This is for servers that are not correctly configured for HA/failover (as far as this software is concerned). A monitored storage server is monitored only; the manager GUI performs no server configuration or management. Chart metrics for filesystem operations will still show for monitored filesystems on the dashboard page.
```
Comment (jgrund) [inline _includes/profile-list.md]: > This is for servers that are not correctly configured for HA/failover (as far as this software is concerned). 

I don't agree with this sentence. This profile is for adding IML to an existing filesystem for monitoring purposes. The HA part is orthogonal.

Comment (johnsonw) [reply _includes/profile-list.md]: For documentation purposes this was copied from the "Creating a new lustre filesystem" page. I will remove this sentence.
